cortix/examples/__init__.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the Cortix toolkit environment
+# https://cortix.org
+
+from cortix.examples.ideal_gas.particle_plotting import Particle_Plot
+from cortix.examples.ideal_gas.particle import Particle
+from cortix.examples.ideal_gas.particle_handler import Particle_Handler
+from cortix.examples.ideal_gas.run_particle import Run_Particle
+
+
+##from cortix.examples.city_justice.prison import Prison
+##from cortix.examples.city_justice.parole import Parole
+##from cortix.examples.city_justice.adjudication import Adjudication
+##from cortix.examples.city_justice.jail import Jail
+##from cortix.examples.city_justice.arrested import Arrested
+##from cortix.examples.city_justice.probation import Probation
+##from cortix.examples.city_justice.community import Community
+
```

Comment:
I don't understand these added lines. The current file in the master branch is this:

https://github.com/dpploy/cortix/blob/4db904d575ec4f8035550713485e686b58fb6cf3/cortix/__init__.py#L1-L14

Did you merge the master into your branch?

Reply:
I can probably merge your changes but I will make changes after to fix a few things.